lstchain/scripts/longterm_dl1_check.py

```diff
@@ -381,13 +474,15 @@ class pixwise_info(tables.IsDescription):
         num_contained_mu_rings_in_run = 0
 
         for subrun in subruns:
-            mufile = 'muons_LST-1.Run{0:05d}.{1:04d}.fits'.format(runnumber, subrun)
+            mufile = Path(args.input_dir,
+                          'muons_LST-1.Run{0:05d}.{1:04d}.fits'.format(
+                                  runnumber, subrun))
 
             dat = None
             try:
                 dat = Table.read(mufile, format='fits')
             except Exception:
-                print('   File', mufile, 'not found - going on')
+                log.warning('File ' + mufile + ' not found - going on')
```

Comment:
Cannot concat `mufile` (Path) as string. Change by `log.warning(f'File {mufile} not found - going on')`

Reply:
True, I had changed that everywhere but here, thanks!